src/main/java/org/cyclonedx/model/Bom.java

```diff
@@ -49,13 +71,15 @@ public void setComponents(List<Component> components) {
         this.components = components;
     }
 
-    public void addComponent(Component component) {
+    public void addComponent(Component comp) {
         if (components == null) {
             components = new ArrayList<>();
         }
-        components.add(component);
+        components.add(comp);
```

Comment:
why changing the name of this parameter?

Reply:
Reverted

Reply:
In: b53fa28